build.cake

```diff
@@ -1,119 +1,32 @@
 // OS X: brew install cake
 // Windows: choco install cake.portable
-//
-// Example:
-// cake -auto=true -openra-root=~/projects/openra
 
 var target = Argument("target", "default").ToLowerInvariant();
 var configuration = Argument("configuration", "Debug");
 
-var rootDir = Directory(".");
-var depsDir = Directory("./OpenRA.Mods.RA2/dependencies");
+Task("default").Does(() => {
```

Comment:
In the longer term we should implement this functionality in the mod template (using the standard makefile / batch / powershell scripts) so that all mods can benefit.